docs/getting-started.md

```diff
@@ -78,7 +78,7 @@ Kibana and the bitnami charts to deploy instances of PostgreSQL and MongoDB.
 If you run the command:
 
 ```bash
-make dev-dracon DRACON_VERSION=v0.13.0
+make dev-dracon DRACON_VERSION=v0.39.0
```

Comment:
Pointing to latest. We need a better way to track this though.

Reply:
if i'm not mistaken, since this is a getting started, shouldn't we make this as easy as possible?
That's why `make install` was created ,it should do this automatically. 
These step by step instructions expose internals which is great if you want to get hands dirty but perhaps they are not suitable for a getting started document anymore?

Reply:
We had a long discussion about this yesterday, the getting started document is too focussed on people who need to have a decent knowledge of Kubernetes, KinD, Helm and make. This is too hands on for a tutorial, we need to have a more automated way that doesn't involve building everything from scratch, and it doesn't make it look like you need to wait 10 minutes for Dracon to do anything because you are actually waiting for Elasticsearch to boot.
The tutorial should be more guided, less hands on and should focus on the results you are getting but give you commands that allow to dig in through the systems to get a better idea of what's happening under the hood.
The documentation for developers which shows how the dev tooling works should be a separate thing

Reply:
Yup, let's discuss this on next week and see what we can do here!

I think for now it's ok if we bump this as we're making sure that folks play with latest - we can decide how to figure this out automatically based on our conversation